platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminAlertImpl.java

```diff
@@ -75,6 +76,15 @@ public AdminAlertImpl(PersistentStore persistentStore, EventAdmin eventAdmin)
       }
     } catch (PersistenceException pe) {
       LOGGER.debug("Error retrieving system alert.", pe);
+
+      return Collections.singletonList(
```

Comment:
❓  Should we provide the admins with any more information to help them possibly debug the issue, such as checking logs/lock files? 
I think this is fine as is, but I'm wondering if we can make this more helpful.

Reply:
I thought about that too. My first thought was to mention Solr, but the system might be using a different persistence implementation. I also thought about mentioned to check the logs, but that seemed like an obvious thing to do. Still, I think it would be fine to mention checking the logs.

Reply:
This might also be the wrong place to make that determination as you are really high on the stack and you wouldn't know why Solr is not reachable yet. It could be because we are initializing, because it went down, because there was a communication issue, ...

Reply:
This alert is not specific to Solr. This message is generated only when the alert notification system cannot retrieve information from PersistentStorage. Persistent storage is presently backed by Solr, but is really an implementation detail. 

The current behavior is to display nothing on the Admin UI if alerts cannot be retrieved from persistent storage. This is disadvantageous to system security because severe alerts available, but because persistent storage is unavailable, nothing is displayed in the admin UI. This PR is meant to solve the "silent failure" situation.

Reply:
That is not how the ticket was written. I would update suggest we re-write the ticket since this PR is not addressing the ticket as written.